config/application.rb

```diff
@@ -1,6 +1,7 @@
 require File.expand_path('../boot', __FILE__)
 
 # Pick the frameworks you want:
+require 'csv'
```

Comment:
is this require necessary? Is Rails not loading the csv? And if it is necessary, can we move the renderer initializer?